docs/security.html

```diff
@@ -320,10 +320,10 @@ <h5>SSL key and certificates in PEM format</h5>
             <pre>listeners</code></pre>
 
             If SSL is not enabled for inter-broker communication (see below for how to enable it), both PLAINTEXT and SSL ports will be necessary.
-            <pre class="line-numbers"><code class="language-text">            listeners=PLAINTEXT://host.name:port,SSL://host.name:port</code></pre>
+            <pre class="line-numbers"><code class="language-text">listeners=PLAINTEXT://host.name:port,SSL://host.name:port</code></pre>
 
             Following SSL configs are needed on the broker side
-            <pre class="line-numbers"><code class="language-text">            ssl.keystore.location=/var/private/ssl/server.keystore.jks
+            <pre class="line-numbers"><code class="language-text">ssl.keystore.location=/var/private/ssl/server.keystore.jks
```

Comment:
looks like we also need to update indent for lines where we have multiple configs/lines. 
<img width="947" alt="Screenshot 2021-02-04 at 6 15 11 PM" src="https://user-images.githubusercontent.com/8134545/106894490-0e7d5100-6715-11eb-9a55-81caf91e187d.png">

Reply:
Nice catch! Updated!